stylelint-polaris/configs/coverage.js

```diff
@@ -189,6 +203,7 @@ module.exports = {
           /--p-range-slider-thumb-size-base/,
           /--p-range-slider-thumb-size-active/,
           /--p-override-visible/,
+          /--p-override-loading-z-index/,
```

Comment:
Moved this legacy custom property from `depth` category to `layout` category since the rest of the `z-index` related rules are categorized there.